### PR TITLE
fix matching

### PR DESCRIPTION
**Pull request by minhtan7**
fix the create form and matching:
- can create new user with name and phone in order to create petition

### CODE REVIEW

**Comment by PrimeTimeTran** [comment]
form
Where might a user create a petition with items? In the receive and donate forms I don't see a button to submit

File: controllers/petitions.controller.js [hunk 1]
```diff
@@ -68,61 +68,75 @@ petitionsController.createWithFund = catchAsync(async (req, res, next) => {
 
 petitionsController.createPetitionWithItems = catchAsync(
   async (req, res, next) => {
-    let { phone, firstName, itemArray, petitionType, description } = req.body;
+    let {
+      phone,
+      firstName,
+      lat,
+      lng,
+      itemArray,
```
Comment (PrimeTimeTran) [inline controllers/petitions.controller.js:76]: I commented with the latest code being tested.

File: controllers/petitions.controller.js [hunk 2]
```diff
@@ -68,61 +68,75 @@ petitionsController.createWithFund = catchAsync(async (req, res, next) => {
...
+      petitionType,
+      description,
+    } = req.body;
     if (!petitionType) {
       return next(new AppError(400, "Required fields are missing!"));
     }
+    console.log(lat, lng);
```
Comment (PrimeTimeTran) [inline controllers/petitions.controller.js]: Could we remove this to keep out logs quieter?

**Review by PrimeTimeTran — APPROVED** [review]

File: controllers/petitions.controller.js (hunk 1 above)
Comment (minhtan7) [reply controllers/petitions.controller.js:76]: this one will work with the lastest fe which I had made the pull request. I see you just merge it so please test it again

File: controllers/petitions.controller.js (hunk 2 above)
Comment (minhtan7) [reply controllers/petitions.controller.js]: ok, thanks, I'll do it

**Pull request merged**